src/reqstool/common/jinja2.py

```diff
@@ -0,0 +1,62 @@
+# Copyright © LFV
+
+import logging
+from importlib.resources import Package, files
+from pathlib import Path
+
+from jinja2 import (
+    BaseLoader,
+    Environment,
+    FileSystemLoader,
+    PackageLoader,
+    Template,
+    TemplateNotFound,
+    select_autoescape,
+)
+
+import reqstool.commands.report.templates
+
+
+class Jinja2Utils:
+
+    @staticmethod
+    def create_template(template_name: str) -> Template:
+        """Returns a Template based on the template name
+
+        Args:
+            template_name (str): The name of the template to retrieve
+
+        Returns:
+            Template: Jinja2 template used for rendering of the AsciiDoc document
+        """
+
+        def load_template(loader: BaseLoader) -> Template:
+            template_env = Environment(
+                loader=loader, autoescape=select_autoescape(), trim_blocks=True, lstrip_blocks=True
+            )
+            return template_env.get_template(template_name)
+
+        try:
+            template_module: Package = reqstool.commands.report.templates
+            template_path: Path = files(template_module).joinpath("")
```

Comment:
Sonar complains about type hit to Path. Should be Traversable

Reply:
Fixed now.